pom.xml

```diff
@@ -61,27 +61,50 @@
         <revision>2.25</revision>
         <changelist>-SNAPSHOT</changelist>
         <java.level>8</java.level>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.235.5</jenkins.version>
```

Comment:
Does it _have_ to be this recent?  What's the justification?

I prefer the minimum version of Jenkins necessary to be as old as possible, but 2.235.5 is only 2020-08-17 and I'd prefer at least a year old, otherwise we're forcing folks to update who may not want to (it happens; being on the latest isn't for everyone).
So, I'd prefer it if we can rebase on 2.190.3 (or earlier) instead - that's (just) over a year old.

Reply:
We can do 2.190 if you want, I based it off of this: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/

The reason to upgrade was the plugin didn't compile on my machine without updating it, also needs to be at least old enough to include the Jenkins bom 2.170 something I think so that testing against super new versions doesn't require hacking in the pom

Reply:
OK, I've read that guidance (just now) and I'd opt for the oldest possible, so 2.190.1 in this case.
I also checked the installed [stats for this plugin](https://stats.jenkins.io/pluginversions/vsphere-cloud.html) and everyone who's on the latest plugin version is on 2.190.1 or later, so making 2.190.1 the minimum Jenkins version this shouldn't prevent anyone using the current latest version of this plugin from upgrading when I do a new release.

...it's also weird that those stats show a plugin version 3.0.3.0 and 3.1.1.0 because there's never (officially) been a 3.x version.  I've no idea where thost stats are coming from; maybe those are unofficial custom builds? ... but I'm not going to lose any sleep over that weirdness - our "lets not upset people" target audience is those who are using the latest version of this plugin.

Reply:
> maybe those are unofficial custom builds

yes exactly, would be nice if they put their org name in there or snapshot as those are filtered out but not much we can do if they publish them with legit looking version